_gsocproposals/2019/proposal_ROOTGlobalModuleIndex.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Implement a GlobalModuleIndex in ROOT and Cling
+layout: gsoc_proposal
+project: ROOT
+year: 2019
+organization:
+  - CERN
+  - University of Lincoln-Nebraska
```

Comment:
Organizations inside proposals have to be referred to with the `organization` attribute, rather than with the `title`. In this case you need to do the following modification:

```suggestion
  - UNL
```